configure.ac

```diff
@@ -49,6 +49,7 @@ AC_PROG_INSTALL
 AC_PROG_CC
 AC_PROG_LIBTOOL
 AM_PROG_AS
+AM_PROG_CC_C_O
```

Comment:
Looks like CentOS 7 (7.2.1511) runs an older automake:
`cmd/zed/Makefile.am:29: warning: compiling 'agents/zfs_diagnosis.c' in subdir requires 'AM_PROG_CC_C_O' in 'configure.ac'`

```
# automake --version
automake (GNU automake) 1.13.4
```